src/intervals/functions.jl

```diff
@@ -167,6 +167,33 @@ function sqrt{T}(a::Interval{T})
     @round(sqrt(a.lo), sqrt(a.hi))  # `sqrt` is correctly-rounded
 end
 
+doc"""
+    pow(x::Interval, n::Integer)
+
+A faster implementation of `x^n` using `power_by_squaring`.
```

Comment:
Thanks! I think it should also be stated that it *may* yield broader intervals than `x^n`.